### PR TITLE
Added page to Logs about permissions and corrected info about permiss…

### PR DESCRIPTION
**Pull request by angelampcosta**
…ions in API.

### CODE REVIEW

File: content/logs/get-started/permissions.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+pcx-content-type: concept
+title: Permissions
+weight: 86
+---
+
+# Permissions
```
Comment (tanushree-sharma) [inline content/logs/get-started/permissions.md:7]: Can the "Permissions" page be moved above "Enable Destinations" in the dropdown?

File: content/logs/get-started/permissions.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Below is a description of the available token and roles permissions for Logs. For information about how to create an API token, please refer to [Creating API tokens](https://developers.cloudflare.com/api/tokens/create/).
```
Comment (tanushree-sharma) [inline content/logs/get-started/permissions.md]: Reword: Below is a description of the available permissions for tokens and roles as they relate to Logs.

File: content/logs/get-started/permissions.md (hunk 1 above)
Comment (angelampcosta) [reply content/logs/get-started/permissions.md:7]: Done.

File: content/logs/get-started/permissions.md (hunk 2 above)
Comment (angelampcosta) [reply content/logs/get-started/permissions.md]: Done.

**Review by tanushree-sharma — APPROVED** [review]

**Review by soheiokamoto — APPROVED** [review]

File: content/logs/get-started/permissions.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+weight: 40
+---
+
+# Permissions
+
+Below is a description of the available permissions for tokens and roles as they relate to Logs. For information about how to create an API token, please refer to [Creating API tokens](https://developers.cloudflare.com/api/tokens/create/).
+
+## Tokens
+
+*   **Logs: Read** - Grants read access logs using Logpull or Instant Logs.
```
Comment (marciocloudflare) [inline content/logs/get-started/permissions.md]:
```suggestion
*   **Logs: Read** - Grants read access to logs using Logpull or Instant Logs.
```

File: content/logs/get-started/permissions.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+*   **Logs: Write** - Grants read and write access to Logpull, Logpush and read access to Instant Logs.
```
Comment (marciocloudflare) [inline content/logs/get-started/permissions.md]:
```suggestion
*   **Logs: Write** - Grants read and write access to Logpull and Logpush, and read access to Instant Logs.
```

File: content/logs/get-started/permissions.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+{{<Aside type="note" header="Note">}}
+For zone scoped datasets, tokens must be zone scoped. For account scoped datasets tokens must be account scoped.
```
Comment (marciocloudflare) [inline content/logs/get-started/permissions.md]:
```suggestion
For zone scoped datasets, tokens must be zone scoped. For account scoped datasets, tokens must be account scoped.
```

File: content/logs/get-started/permissions.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+The **Administrator Read only** and **Log Share Reader** roles only have access to Instant Logs and Logpull. This role does not have permissions to view the configuration of Logpush jobs.
+
+### Assigning or removing a role 
+
+To check the list of members in your account, or to manage roles and permissions:
+
+1.  Navigate to the Cloudflare Dashboard (https://dash.cloudflare.com/login).
+2.  From your Account Home, navigate to Members.
+3.  Enter a member’s email address to add them to your account, and click Invite.
+4.  Alternatively, scroll down to the Members card to find a list of members with their status and role.
```
Comment (marciocloudflare) [inline content/logs/get-started/permissions.md]:
```suggestion
1.  Navigate to the Cloudflare Dashboard (https://dash.cloudflare.com/login).
2.  From your Account Home, navigate to **Members**.
3.  Enter a member’s email address to add them to your account, and click **Invite**.
4.  Alternatively, scroll down to the **Members** card to find a list of members with their status and role.
```

Comment (marciocloudflare) [reply content/logs/get-started/permissions.md]: I feel like there's a step missing from 1. to 2. In the dash I don't see any `Members`, just the accounts I can click to select. Should step 1 be `Navigate to the Cloudflare Dashboard (https://dash.cloudflare.com/login) and select your account`?

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**